### ISSUE TITLE
Weekday 1-5 means Su-Th instead

### ISSUE THREAD

**Issue by erichirlemann** [ncb000gt/node-cron#181]
Trying to as 1-5 as Mo-Fr does not work as expected, the task gets triggered on Sunday but not on Friday. From https://github.com/padolsey/cron.js , it seems that  one must put 2-6 for Mo-Fr instead.
This is an unexpected behavior regarding the usual Unix crontab standards.

**Comment by ncb000gt** [comment]
Are you sure about this? I'm looking at the code which aliases mon-fri as 1-5. Also, the ranges start at 0 and go to 6, so the only thing that is unix cron related that shouldn't be supported is that 0 and 7 are both sunday.

I'm guessing you are seeing differently?

Do you have a code example I can use to create a test?

**Comment by erichirlemann** [comment]
Yes, I'm sure, but I realized my piece of code was inheriting from version 0.3.3 of the cron package. I apologized, I should have check this point first.

To reproduce the issue, I used this script:
https://gist.github.com/erichirlemann/08ca10d50fc557913cdd

With version 0.3.3, the trace looks like:
`
everyMinute06 started
everyMinute17 started
Cronjob 1-7 run at Tue Dec 01 2015 17:53:00 GMT+0100 (CET)
Cronjob 1-7 run at Tue Dec 01 2015 17:54:00 GMT+0100 (CET)
`

With version 1.0.9, the trace looks like:
`
everyMinute06 started
everyMinute17 started
Cronjob 0-6 run at Wed Dec 02 2015 13:49:00 GMT+0100 (CET)
Cronjob 0-6 run at Wed Dec 02 2015 13:50:00 GMT+0100 (CET)
`
what is the expected trace.
